tests/Maker/FunctionalTest.php

```diff
@@ -208,6 +258,17 @@ public function getCommandTests()
             ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeFunctional')
         ];
 
+        yield 'functional_in_custom_root_namespace' => [MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeFunctionalTest::class),
+            [
+                // functional test class name
+                'FooBar',
+            ])
+            ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeFunctionalInCustomRootNamespace')
+            ->changeRootNamespace('Custom')
+            ->addPreMakeCommand('composer dump-autoload')
+        ];
+
```

Comment:
I think we only really need to test one case, because each maker is using the exact same code to place files. Well, maybe 2 cases: `MakeCommand` (as a general case) and `MakeEntity`, because it *is* indeed a bit more complex.

Reply:
It is. I will remove useless tests